src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs

```diff
@@ -78,6 +82,24 @@ public void StartTimer()
             }
         }
 
+        /// <summary>
+        /// This will post an event containing the information about patching on CoordinatorService.
+        /// </summary>
+        private void PostWUUpdateEventOnServicePackage()
+        {
+            CheckpointFileData fileData = this.ReadCheckpointFile();
+            string formatString = "LastAttemptedWUTime : {0}, NextWUDownloadTime : {1}";
```

Comment:
>LastAttemptedWUTime [](start = 35, length = 19)

Changed to Last patching attempt happened at {0}, Next patching cycle is scheduled at {1}.